recipes/itk/all/conanfile.py

```diff
@@ -0,0 +1,250 @@
+import os
+import re
+
+from conans import ConanFile, CMake, tools
+
+class LibITKConan(ConanFile):
+    name = "itk"
+
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```
This will solve your problem related to zlib